src/Skills.jsx

```diff
@@ -161,6 +161,9 @@ class Skills extends Component {
          },
        });
      })
+     .then(() => {
+       browserHistory.replace('/skills/');
```

Comment:
If you push then that URL will still be in the browser history, which does not make sense since it's been deleted and therefore trying to visit that URL again (say, if you press the back button) would result in a 404.